package.json

```diff
@@ -155,9 +155,15 @@
         "typescript": "^4.8.2",
         "vsce": "^2.11.0"
     },
+    "extensionDependencies": [
+        "viper-admin.viper"
```

Comment:
He already uses `$VIPER_SERVER`! But the ideal thing would be to frob the port number form the `viper-ide` panel, and connect to the already running server.

Reply:
This would be ideal. The main challenge here is that VS Code loads extensions based on [activation events](https://vscode-docs.readthedocs.io/en/stable/extensionAPI/activation-events/), so it's currently required to manually open a `*.vpr` file to activate the Viper LS. 

Update: I just found a way to programmatically activate the Viper extension using the [extensions API](https://code.visualstudio.com/api/references/vscode-api#extensions), so now we just need a way to reliably retrieve the port number.